File: SmartCom/SmartCom/Models/Order.cs
using System;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;

//класс заказа
namespace SmartCom.Models
{
    public class Order
    {
        [ScaffoldColumn(false)]
        [Key]
        [DatabaseGeneratedAttribute(DatabaseGeneratedOption.Identity)]
        public int OrderID { get; set; }

        public string Title { get; set; }

        public int? ItemID { get; set; }

        public int? Discount { get; set; }

        public int Count { get; set; }

        public int Price { get; set; }
    
        public string Code { get; set; }

        public string Status { get; set; }

        public DateTime OrderDate { get; set; }

        public DateTime ChangeStatus { get; set; }

        public string UserName { get; set; }

    }
}

File: SmartCom/SmartCom/app/model/Order.js
//модель заказов
Ext.define('ExtMVC.model.Order', {
    extend: 'Ext.data.Model',
    idProperty: 'OrderID',

    fields: [
        { name: 'OrderID', type: 'int' },
        { name: 'ItemID', type: 'int' },
        { name: 'Title', type: 'string' },
        { name: 'Code', type: 'string' },
        { name: 'Status', type: 'string' },
        { name: 'Count', type: 'int' },
        { name: 'Price', type: 'int' },
        { name: 'Discount', type: 'int'},
        { name: 'OrderDate', type: 'date', dateFormat: 'MS' },
        { name: 'ChangeStatus', type: 'date', dateFormat: 'MS' },
        { name: 'UserName', type: 'string' }
    ],

});